tests/e2e/utils/helpers.go

```diff
@@ -70,6 +79,19 @@ func SubnetConfigExists(subnetName string) (bool, error) {
 	return genesisExists && sidecarExists, nil
 }
 
+func APMConfigExists(subnetName string) (bool, error) {
```

Comment:
this is check for subnet sidecar to exist. maybe can be renamed, and also reused in the other functions here

Reply:
I reworked this so sidecarExists and genesisExists are separate functions and this just wraps that function.